tests/public_data/test_sql/test/incremental_query_v1/metadata.yaml

```diff
@@ -1,6 +1,5 @@
-friendly_name: Test table for an incremental query
```

Comment:
Turns out that writing it this way will result in a newline character when parsing using `yaml`